mache-vertx/src/main/groovy/com/excelian/mache/vertx/CassandraConfig.groovy

```diff
@@ -0,0 +1,32 @@
+package com.excelian.mache.vertx;
+
+import com.excelian.mache.core.SchemaOptions
+import com.datastax.driver.core.Cluster
+
+import static com.excelian.mache.cassandra.builder.CassandraProvisioner.cassandra
+import static com.excelian.mache.guava.GuavaMacheProvisioner.guava;
+import static com.excelian.mache.builder.MacheBuilder.mache;
+
+public class CassandraConfig {
+    protected static String keySpace = "NoSQL_Nearside_Test_" + new Date().toString();
+
+    public static void main(String[] args) {
+        MacheRestService restService = new MacheRestService();
+
+        restService.run({ ->
+            mache(String.class, String.class)
+                    .cachedBy(guava())
+                    .storedIn(
+                    cassandra()
+                            .withCluster(Cluster.builder()
+                                .withClusterName("BluePrint")
+                                .addContactPoint("192.168.3.4")
+                                .withPort(9042))
+                            .withKeyspace(keySpace)
+                            .withSchemaOptions(SchemaOptions.CREATE_AND_DROP_SCHEMA)
```

Comment:
I have updated this to use a test table as we create/drop in the keyspace. As an aside with JSON the builder ends with .withIDField instead of .build, is that intentional?